### ISSUE TITLE
Broken metaboxes conditional logic.

### ISSUE THREAD

**Issue by HeroBanana** [htmlburger/carbon-fields#1093]
You broke the metaboxes in this commit:
https://github.com/htmlburger/carbon-fields/commit/59e7fbd45119aa419b9bc0d5e022ab4042aa3db6

Please change back to:
```
export default compose(
	applyWithSelect,
	applyWithEffects,
)( ConditionalDisplay );
```

**Comment by ridinghoodmedia** [comment]
Hmm is this plugin abandoned..

**Comment by HTMLBurger-NG** [comment]
This issue was fixed in the current release (v3.5.0).
Please update the WordPress core and Carbon Fields to the newest version and let us know if the issues still exist.
